src/python/CMSMonitoring/openstack_accounting.py

```diff
@@ -0,0 +1,172 @@
+                          'maxServerGroupMembers': Use(int),
+                          'totalRAMUsed': Use(int),
+                          'totalCoresUsed': Use(int),
+                          'totalInstancesUsed': Use(int),
+                          'totalFloatingIpsUsed': Use(int),
+                          'totalSecurityGroupsUsed': Use(int),
+                          'totalServerGroupsUsed': Use(int),
+                          'maxTotalVolumes': Use(int),
+                          'maxTotalSnapshots': Use(int),
+                          'maxTotalVolumeGigabytes': Use(int),
+                          'maxTotalBackups': Use(int),
+                          'maxTotalBackupGigabytes': Use(int),
+                          'totalVolumesUsed': Use(int),
+                          'totalGigabytesUsed': Use(int),
+                          'totalSnapshotsUsed': Use(int),
+                          'totalBackupsUsed': Use(int),
+                          'totalBackupGigabytesUsed': Use(int),}])
+
+
+SUMMARY_COL_ORDER = {'openstackProjectName': 'Openstack Project Name',
+                     'maxTotalInstances': 'maxTotalInstances',
+                     'maxTotalCores': 'maxTotalCores',
+                     'maxTotalRAMSize': 'maxTotalRAMSize',
+                     'maxSecurityGroups': 'maxSecurityGroups',
+                     'maxTotalFloatingIps': 'maxTotalFloatingIps',
+                     'maxServerMeta': 'maxServerMeta',
+                     'maxImageMeta': 'maxImageMeta',
+                     'maxPersonality': 'maxPersonality',
+                     'maxPersonalitySize': 'maxPersonalitySize',
+                     'maxSecurityGroupRules': 'maxSecurityGroupRules',
+                     'maxTotalKeypairs': 'maxTotalKeypairs',
+                     'maxServerGroups': 'maxServerGroups',
+                     'maxServerGroupMembers': 'maxServerGroupMembers',
+                     'totalRAMUsed': 'totalRAMUsed',
+                     'totalCoresUsed': 'totalCoresUsed',
+                     'totalInstancesUsed': 'totalInstancesUsed',
+                     'totalFloatingIpsUsed': 'totalFloatingIpsUsed',
+                     'totalSecurityGroupsUsed': 'totalSecurityGroupsUsed',
+                     'totalServerGroupsUsed': 'totalServerGroupsUsed',
+                     'maxTotalVolumes': 'maxTotalVolumes',
+                     'maxTotalSnapshots': 'maxTotalSnapshots',
+                     'maxTotalVolumeGigabytes': 'maxTotalVolumeGigabytes',
+                     'maxTotalBackups': 'maxTotalBackups',
+                     'maxTotalBackupGigabytes': 'maxTotalBackupGigabytes',
+                     'totalVolumesUsed': 'totalVolumesUsed',
+                     'totalGigabytesUsed': 'totalGigabytesUsed',
+                     'totalSnapshotsUsed': 'totalSnapshotsUsed',
+                     'totalBackupsUsed': 'totalBackupsUsed',
+                     'totalBackupGigabytesUsed': 'totalBackupGigabytesUsed'}
```

Comment:
I suggest to separate words with space instead of using camel-case column names. This is a good example that you did `openstackProjectName': 'Openstack Project Name'`. 
Because, JS `white-space: wrap;` option can break column words and put them on top of each other, so table width decreases.